README.md

```diff
@@ -157,6 +156,95 @@ Working in progress....
+
+### Sign-in page
+
+The sign-in page is accessible to all visitors, allowing them to log into the website. Additionally, this page offers links that guide new users to the sign-up page and provides a redirect to the 'Forgot Password' page for users who need to recover their login credentials.
+
+![Sign in page](./docs/images/wireframes/sign-in-page.png)
+
+### Sign up page
+
+The Sign-up page enables new visitors to create an account and includes a link for existing users to return to the Sign-in page.
+
+![Sign up page](./docs/images/wireframes/sign-up-page.png)
+
+### Forgot password page
+
+Users will be directed to the Forgot Password page after clicking the 'Forgot Password' link. They are required to enter their email address and click the 'Send' button to receive a link for resetting their password.
+
+![Forgot Password page](./docs/images/wireframes/forgot-password-page.png)
+
+### Reset password page
+
+Upon clicking the reset password link received via email, users will be taken to a page where they can establish a new password.
+
+![Reset Password page](./docs/images/wireframes/set-new-password-page.png)
+
+### Search page
+
+The search bar enables users to refine their product search using keywords or by selecting from the categories in the drop-down menu. The resulting matches will be presented on the search results page.
+
+![Search page](./docs/images/wireframes/search-page.png)
+
+### Product details page
+
+Clicking on a product image or name will direct users to the Product Details page, where they can find more information about the product and add it to their cart if they wish to make a purchase.
+
+![Product Details page](./docs/images/wireframes/product-details-page.png)
+
+### Shopping cart page
+
+When users click the cart icon in the header, they will be taken to the Cart page. Here, they can view all the products they have added and can choose to proceed with their purchase by clicking the 'Proceed to checkout' button or continue shopping by clicking the 'Continue shopping' button.
+
+![Shopping Cart Page](./docs/images/wireframes/shopping-cart-page.png)
+
+### Checkout page
+
+Users intent on proceeding to payment must log in or register an account to be taken to the Checkout page after selecting the 'Proceed to Checkout' button on the cart page. They are required to provide delivery address details and credit card information, and must review and confirm the order summary to finalize the payment process.
+
+![Checkout Page](./docs/images/wireframes/checkout-page.png)
+
+### User account page
+
+Authenticated users have access to their personal accounts where they can view their profiles and purchase history. Additionally, they have the option to change their current password in this section.
+
+![User Account Page](./docs/images/wireframes/user-account-page.png)
+
+### Admin dashboard page
+
+The admin dashboard is organized into four tabbed pages for streamlined navigation:

```

Comment:
Can you change this part `The admin dashboard is organized into four tabbed pages` to:

`The admin dashboard has four pages, which are structured as tabs....`